site/src/_pages/docs/2_publishing-blocks.mdx

```diff
@@ -24,9 +24,7 @@ The JSON metadata file should look like this:
   branch: "main",
   // REQUIRED - name of the folder your built blocks will appear in after running `yarn build`
   distDir: "dist",
-  // IF NEEDED - where your commands are in a subfolder,
-  // provide ONE OF the folder path or yarn workspace
-  folder: "packages/blocks/block-embed",
+  // OPTIONAL - modify yarn commands
```

Comment:
This looks like an unintended change from resolving merge conflicts - if that's right can you revert it?

Reply:
Yup, it was part of a merge conflict. Reversed now.